### ISSUE TITLE
Questionable data in Travis-CI log

### ISSUE THREAD

**Issue by josephwright** [latex3/latex2e#20]
[ Copied from https://github.com/latex3/l3build/issues/55 ]

Looking into the travis log after uploading some code to latex2e I saw the following section:

```
Running script build.lua with target "check" for module doc
usage: build.lua <command> [<options>] [<names>]
The most commonly used l3build commands are:
   check      Run all automated tests
   clean      Clean out directory tree
   ctan       Create CTAN-ready archive
   doc        Typesets all documentation files
   install    Installs files into the local texmf tree
   save       Saves test validation log
   setversion Update version information in sources
   uninstall  Uninstalls files from the local texmf tree
   unpack     Unpacks the source files into the build tree
Valid options are:
   --config|-c        Sets the config(s) used for running tests
   --date             Sets the date to insert into sources
   --dry-run          Dry run for install
   --engine|-e        Sets the engine(s) to use for running test
   --epoch            Sets the epoch for tests and typesetting
   --first            Name of first test to run
   --force|-f         Force tests to run if engine is not set up
   --halt-on-error|-H Stops running tests after the first failure
   --last             Name of last test to run
   --pdf|-p           Check/save PDF files
   --quiet|-q         Suppresses TeX output when unpacking
   --rerun            Skip setup: simply rerun tests
   --shuffle          Shuffle order of tests
   --texmfhome        Location of user texmf tree
   --version|-v       Sets the version to insert into sources
See l3build.pdf for further details.
Running script build.lua with target "check" for module required/cyrillic
```
might be nicer if the `build.lua` in the latex2e/doc directory has a "check" target or that the the target is not called for that module or that we find a way to avoid these handwritten build files in the first place.

**Comment by josephwright** [comment]
As noted in https://github.com/latex3/l3build/issues/55, this is due to a 'local' `build.lua` file: I'll fix it.

**Comment by josephwright** [comment]
Should be fixed by 01e5d937d54a3a5adfdf3348291078780776161b
